Team-C/test_wizard/.github/workflows/dart-and-flutter-analysis.yml

```diff
@@ -0,0 +1,30 @@
+name: Dart
+
+on:
+  push:
+    branches: ['developer']
```

Comment:
I think it might be due to this line, change this to your branch?

Reply:
try adding a line to a fine and push that to attempt to trigger the action?

Reply:
I think it's just because it isn't actually part of the codebase yet, which is what the PR would fix. Using a CI/CD Runner like `act` wasn't working due to my chip architecture (M1, ARM64) being different than what the flutter action (linux, amd64 if I understand the docs correctly) was expecting. Merging now to figure it out and then I'll update if necessary.